ext/curl/config.w32

```diff
@@ -21,16 +21,14 @@ if (PHP_CURL != "no") {
 		(((PHP_ZLIB=="no") && (CHECK_LIB("zlib_a.lib;zlib.lib", "curl", PHP_CURL))) ||
 			(PHP_ZLIB_SHARED && CHECK_LIB("zlib.lib", "curl", PHP_CURL)) || (PHP_ZLIB == "yes" && (!PHP_ZLIB_SHARED))) &&
 		!isNaN(ver_num) &&
-		(ver_num <= parseInt("0x073b00") || ver_num > parseInt("0x073b00") &&
-			CHECK_LIB("normaliz.lib", "curl", PHP_CURL) &&
-			CHECK_LIB("libssh2.lib", "curl", PHP_CURL) &&
-			CHECK_LIB("nghttp2.lib", "curl", PHP_CURL))
```

Comment:
That was supposed to be:
````
		(ver_num <= parseInt("0x073b00")) || (ver_num > parseInt("0x073b00") &&
			CHECK_LIB("normaliz.lib", "curl", PHP_CURL) &&
			CHECK_LIB("libssh2.lib", "curl", PHP_CURL) &&
			CHECK_LIB("nghttp2.lib", "curl", PHP_CURL))

````
But see #16787 for why I agree that dropping the check is fine.

Reply:
I see. Ok. Yes, there was also a comment about this change on the initial commit but never got addressed back then in 2018. Then fine to be removed.